tests/common/devices.py

```diff
@@ -1686,6 +1692,10 @@ def run_redis_cli_cmd(self, redis_cmd):
     def get_ip_route_info(self, dstip):
         return self.sonichost.get_ip_route_info(dstip, self.cli_ns_option)
 
+    @property
+    def os_version(self):
+        return self.sonichost.os_version
```

Comment:
Why need to define this property? The `duthost` is an instance of `MultiAsicSonicHost`. Accessing `duthost.os_version` will eventually get `SonicHost.os_version`.
```
(Pdb) duthost
<tests.common.devices.MultiAsicSonicHost object at 0x7f7d667fcbd0>
(Pdb) duthost.os_version
u'master.606-3acf7006'
```

Reply:
This is from AsicHost. AsicHost didn't define this attribute causes access exception.

Reply:
But this SonicAsic.os_version property is not accessed in this PR's code change, right?

Reply:
It is accessed by checks.py. The multi-asic refactoring causes the show_interface() being called from asichost level.

Reply:
Ah, I see, the `dut` argument is an instance of duthost.asics. I was confused by its name.